src/screens/landingPageScreens/ApplicationInfoScreen.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import BlankImg from "/images/Landing Page Images/BlankImage.png";
+
+interface CardData {
+    title: string;
+    description: string;
+}
+
+interface CardProps {
+    title: string;
```

Comment:
same as above, move this to util folder

interface CardDataProps {
    title: string;
    description: string;
    isOpen: boolean;
    onClick: () => void;
}